scenarios/node_hog_engine.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+  namespace: default
+spec:
+  # It can be true/false
+  annotationCheck: 'false'
+  # It can be active/stop
+  engineState: 'active'
+  chaosServiceAccount: node-cpu-hog-sa
+  monitoring: false
+  # It can be delete/retain
+  jobCleanUpPolicy: 'delete'
+  experiments:
+    - name: node-cpu-hog
+      spec:
+        components:
+          env:
+            # set chaos duration (in sec) as desired
+            - name: TOTAL_CHAOS_DURATION
+              value: '60'
+
+             # ENTER THE NAME OF THE APPLICATION NODE
+            - name: APP_NODE
```

Comment:
I think this env var needs to be TARGET_NODES (see: https://docs.litmuschaos.io/docs/node-cpu-hog/#prepare-chaosengine)   @jordigilh Can you confirm?   Even when I use TARGET_NODES, though, it is not honored and the load runs on a different node.

Reply:
It looks like the key was updated between versions 1.9 and 1.10. In 1.9 documentation it shows APP_NODES still (https://docs.litmuschaos.io/docs/1.9.0/node-cpu-hog/) 

Seems like the versions are updated quite often. Do we want to just pick a version and keep using it until we have a need to update. Should we use 1.10 now since it's the latest?

Reply:
@paigerube14 confirmed it should be [named](https://github.com/litmuschaos/litmus-go/blob/master/pkg/generic/node-cpu-hog/environment/environment.go#L32)  `TARGET_NODES`.
They also changed the structure of the probes in the schema. Since you don't use probes in this case, you should be good to move to 1.10.0.

Reply:
Updated config file to use the 1.10.0 versions.